### DISCUSSION TITLE
Sunny Island Temperature

### DISCUSSION THREAD

**Discussion by teleplague** [SBFspot/SBFspot#680] [General]
Hi. Thanks for the awesum product.

I've been using SBFspot to read values from a STP15000, and send it to our openhab server.

We also have 3x Sunny Islands, and we recently had an issue where one of the island got extremely hot.

On the tripower I can read the value "InvTemperature" to get the temperature of the tripower.

I thought I'll do the same to read the temp of the Master Island.
SBFspot connects to the island, and gets the serial nr etc. but I get "getInverterTemperature returned an error: 21" when trying to read temperature.

I dont know if its possible to read island temperature, or if I need to set a different value other than "InvTemperature"

Any help would be appreciated.

**Comment by sbf-** [comment]
Unfortunately, not all devices report their internal temperature. Maybe it is possible but it is not supported by SMA (e.g. SunnyExplorer is not showing it)

Error 21 means "Item Not Available"

**Reply by teleplague** [reply]
I thought this might be the case. Thank you for your input!
